fsw/src/cs_events.h

```diff
@@ -60,7 +60,7 @@
  *
  *  This event message is issued when a reset counters command has been received.
  */
-#define CS_RESET_DBG_EID 3
+#define CS_RESET_INF_EID 3
```

Comment:
Why does this event name change from using DBG to INF? 

Line 57 may need an update to indicate this event is informational type.

Reply:
Should be updated now Justin.